src/scripts/clientUtils.js

```diff
@@ -29,20 +29,43 @@ export async function serverAddress() {
     // if (current) return Promise.resolve(current);
 
     const urls = [];
+    if (window.location.href.indexOf('/web/') !== -1) { // Do we polyfill String.prototype.includes?
```

Comment:
```suggestion
    if (window.location.href.indexOf('/web/') !== -1) {
```
I don't see babel-polyfill added in the webpack config so probably not

Reply:
It should be covered by the corejs import... babel-polyfill is deprecated now